fuzzer/modules/sql_injection/strategies/lib/postgres.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
postgres utility lib to help with managing db state.

Reply:
We don't actually need this as part of this PR. Sorry this is a bit messy to include in a PR, but 🤷